tests/features/warmshowers/create_account.feature

```diff
@@ -0,0 +1,55 @@
+#language:en
+
+Feature: I can create an account
+	In order to become a WarmShowers member
+	As a new user
+	I can create an account
+
+Scenario Outline: As a new user, I can view the account creation page in my language
+	Given I am at the <page> page
+	And I have selected <lang> language
+	When I see the Sign Up link in my language
+	And I click the <size> Sign Up link in my chosen language
+	Then I see the User Account page in my chosen language
+	And the Create New Account tab in my chosen language
+
+	Examples:
+		|page			|lang 		|size	|
+		|landing		|English	|large	|
+		|landing		|English	|small	|
+		|FAQ			|English	|large	|
+		|Forums			|English	|small	|
+		|landing		|Francais	|large	|
+		|landing		|Espanol	|small	|
+
+#Need to edit these examples with longer "about" text and proper dummy accounts.
+Scenario Outline: As a new user, I can complete and submit the registration form
+	Given I am at the User Account page
+	And on the Create New Account tab
+	When I enter <username> in the Username field
+	And <email> in the email address field
+	And reenter the correct email address in the confirm email address field
+	And enter <password> in the password field
+	And reenter the correct password in the confirm password field
+	And select a <country>
+	And enter a <city>/town
+	And enter a <state>/province
+	And enter a <fullname>
+	And enter at least 15 words of <aboutText>
+	Then I should see a modal with "A validation e-mail has been sent to your e-mail address. In order to gain full access to the site, you will need to follow the instructions in that message."
+	And I should see the New Member Validation Instructions page
+	And I should receive a validation email
+
+	Examples:

```

Comment:
This is spot on, just a note on the level of detail. I think the best way to think of these initial feature tests is as guides for manual testing.

So... each time we deploy a new piece of functionality (in this case the site upgrade) we want to run these tests, manually. In this case I don't think we want to be creating four accounts, the shorter it takes to test the better, if we can get it down to even a single one while still having a good level of coverage that would be ideal.